task_spec/spec/acceptance/init_spec.rb

```diff
@@ -38,6 +38,19 @@ def log_output_errors(result)
     puts logger.info(out)
   end
 
+  # Added this method to simplify the 'case' condition
+  # used for target_platform, which will use latest puppet_agent
+  # in below mentioned test spec
+  def latest_platform_list
+    %r{
+      el-9-aarch64|
+      ubuntu-22.04-aarch64|
```

Comment:
```suggestion
      ubuntu-22\.04-aarch64|
```

Reply:
Thank you @kenyon. Good catch.

I have modified the regex. Cheers!!